### PR TITLE
Create UserValidator

### PR DESCRIPTION
**Pull request by Sedarius-1**

### CODE REVIEW

File: user/src/main/java/org/shareio/backend/core/model/UserValidator.java [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+
+    public static void validateUser(UserProfileGetDto userProfileGetDto) throws MultipleValidationException {
+        Map<String, String> errorMap = new HashMap<>();
+        try{
+            validateName(userProfileGetDto.name());
+        } catch(ValidationException validationException){
+            errorMap.put("Name", validationException.getMessage());
+        }
+
+        //TODO: CONTINUE
```
Comment (Tomunek) [inline user/src/main/java/org/shareio/backend/core/model/UserValidator.java:22]: Yeah
Continue

Comment (Sedarius-1) [reply user/src/main/java/org/shareio/backend/core/model/UserValidator.java:22]: Task was to create it, not finish it

**Review by Tomunek — CHANGES_REQUESTED** [review]
Continue

**Comment by Sedarius-1** [comment]
No point now, let me push whatever else is left and then I finish

**Review by Tomunek — APPROVED** [review]

**Pull request merged**